src/UI/words.tsx

```diff
@@ -40,6 +40,8 @@ const dict = {
   remove: "Remove",
   username: "Username",
   password: "Password",
+  "load.previous": "Load older",
```

Comment:
Load previous sounds nicer to me.

Reply:
That was my initial attempt, but Bart  opted for that one, I will ask though, because previously I had load newer in the copy